app/assets/javascripts/application.js

```diff
@@ -427,3 +427,10 @@ function setPowerState(item, status){
   power_actions.hide();
   $('[rel="twipsy"]').tooltip();
 }
+
+function isSecurePage() {
+  if (location.protocol === 'https:')
```

Comment:
lol... it is also used by ```$.cookie($.cookieName, JSON.stringify($.foremanSelectedHosts), { secure: isSecurePage() });``` which jumps to another file :bowtie: 
But I understand what you're saying, will refactor